dev.js

```diff
@@ -12,14 +12,20 @@ const {
       short: 'c',
     },
     externalClasses: {
-      type: 'boolean',
+      type: 'string',
```

Comment:
I see that it is used like this:
` const safeExternalClasses = options.externalClasses || [];`

should it be `string[]`?